src/marlin/marlin_pc/mod.rs

```diff
@@ -390,8 +279,6 @@ where
             let challenge_j = opening_challenges(opening_challenge_counter);
             opening_challenge_counter += 1;
 
-            assert_eq!(degree_bound.is_some(), rand.shifted_rand.is_some());
```

Comment:
Why was this assertion removed?

Reply:
likely also due to incorrect sync, let me also take a look